src/xrScriptEngine/script_engine.cpp

```diff
@@ -524,7 +524,7 @@ struct raii_guard : private Noncopyable
 #ifdef DEBUG
             static const bool break_on_assert = !!strstr(Core.Params, "-break_on_assert");
 #else
-            static const bool break_on_assert = true;
+            static const bool break_on_assert = false;	// xxx: true will hide real lua error, LuaJIT problem?
```

Comment:
Indeed. I'm starting to forget things I was working on :)
So, given that we use new LuaJIT, is there a point of having `break_on_assert` thing?